### ISSUE TITLE
Handler 'systemd reload' frequently times out

### ISSUE THREAD

**Issue by gderber** [bloodymage/ansible-collection-autonomy#368]
```
RUNNING HANDLER [bloodymage.autonomy.collection_handlers : Force systemd to reread configs] *************************************************************************************************
fatal: [host1]: FAILED! => {"changed": false, "msg": "failure 1 during daemon-reload: Failed to reload daemon: Method call timed out\n"}
fatal: [host2]: FAILED! => {"changed": false, "msg": "failure 1 during daemon-reload: Failed to reload daemon: Method call timed out\n"}
fatal: [host3]: FAILED! => {"changed": false, "msg": "failure 1 during daemon-reload: Failed to reload daemon: Method call timed out\n"}
```

This happens a lot, different hosts, but same results.  Unfortunately, it's not consistent.

**Comment by gderber** [comment]
Looks like this may have been caused by a missing 'become: yes' as part of the task.

**Comment by gderber** [comment]
Fixed with commit 4907ac5
